src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -12,466 +14,121 @@ namespace System.Tests
     public class Perf_Double
     {
         public static readonly double dValue = 1.23456789E+5;
```

Comment:
Any reason this is a static readonly rather than a const?  Can it be private, and if it remains a static, prefixed with s_?